pkg/relayer/session/claim.go

```diff
@@ -301,19 +313,27 @@ func (rs *relayerSessionsManager) payableProofsSessionTrees(
 	for _, sessionTree := range sessionTrees {
 		// If the supplier operator can afford to claim the session, add it to the
 		// claimableSessionTrees slice.
-		if supplierOperatorBalanceCoin.IsGTE(*proofSubmissionFeeCoin) {
+		if supplierOperatorBalanceCoin.IsGTE(claimAndProofSubmissionCost) {
```

Comment:
Can you move this into a local constant called

supplierCanAffordClaimAndProofFees := supplierOperatorBalanceCoin.IsGTE(claimAndProofSubmissionCost)